api/server_test.go

```diff
@@ -34,15 +35,16 @@ var (
 // serverTester contains a server and a set of channels for keeping all of the
 // modules synchronized during testing.
 type serverTester struct {
-	cs      *consensus.ConsensusSet
-	gateway modules.Gateway
-	host    modules.Host
-	hostdb  modules.HostDB
-	miner   modules.Miner
-	renter  modules.Renter
-	tpool   modules.TransactionPool
-	exp     modules.Explorer
-	wallet  modules.Wallet
+	cs        *consensus.ConsensusSet
```

Comment:
ah yeah. I think there are still a few of those floating around, but it's not necessary anymore outside of the consensus package.